docs/index.md

```diff
@@ -23,14 +23,16 @@ Documentation is an important part of this project, and we take the same open-so
 
 If there’s a particular area of documentation that you’d like to see that’s missing, please [file a bug](https://github.com/canonical/synapse-operator/issues).
 
-# Contents
-
-1. [Tutorial](tutorial)
-  1. [Getting Started](tutorial/getting-started.md)
-1. [How to](how-to)
-  1. [Contribute](how-to/contribute.md)
-1. [Reference](reference)
-  1. [Actions](reference/actions.md)
-  1. [Integrations](reference/integrations.md)
-1. [Explanation](explanation)
-  1. [Charm architecture](explanation/charm-architecture.md)
```

Comment:
this is intended in git, it gets translated to the navigation table on discourse btw